### PR TITLE
[README] Remove hidden content and file duplicate

### PR DESCRIPTION
**Pull request by sbalandi**
- also resized/align some gifts/images

### CODE REVIEW

**Comment by eaidova** [comment]
@sbalandi Great work! I think the last part "You will have a lot of fun with this section:" should be removed as we make whole tutorials list visible, now no need to add focus on specific notebooks

**Review by adrianboguszewski — COMMENTED** [review]
@sbalandi Is markdown replaced by HTML in some readme files for faster loading?

**Comment by sbalandi** [comment]
> @sbalandi Great work! I think the last part "You will have a lot of fun with this section:" should be removed as we make whole tutorials list visible, now no need to add focus on specific notebooks

done

File: notebooks/110-ct-segmentation-quantize/README.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 
-![kidney segmentation animation](https://user-images.githubusercontent.com/77325899/154279555-aaa47111-c976-4e77-8d23-aac96f45872f.gif)
+<p align="center">
+    <img src="https://user-images.githubusercontent.com/77325899/154279555-aaa47111-c976-4e77-8d23-aac96f45872f.gif"/>
+</p>
 
 ## Notebook Contents
 
-This folder contains five notebooks that show how to train,
+This folder contains five notebooks that show how to train,C
```
[the remark this answers is not in the document]
Comment (sbalandi) [reply notebooks/110-ct-segmentation-quantize/README.md]: removed

I replaced it to align pictures and control its size

**Review by eaidova — APPROVED** [review]

**Pull request merged**